php/app/src/Comments/Application/UseCase/DeleteTaskCommandHandler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Comments\Application\UseCase;
+
+use App\Comments\Application\Command\DeleteCommentsCommand;
+use App\Comments\Domain\Repository\CommentsRepositoryInterface;
+use NinjaBuggs\ServiceBus\Command\UseCaseInterface;
+use NinjaBuggs\ServiceBus\TransactionalHandlerInterface;
+
+class DeleteTaskCommandHandler implements UseCaseInterface, TransactionalHandlerInterface
```

Comment:
DeleteTaskCommandHandler using CommentsRepositoryInterface $usersRepository  to delete $user, funny